### PR TITLE
Modern times

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

File: test/simple_load.erl [hunk 1]
```diff
@@ -1,72 +0,0 @@
--module(simple_load).
```
Comment (paulo-ferraz-oliveira) [inline test/simple_load.erl:1]: This file is unused.

File: rebar.config [hunk 2]
```diff
@@ -1,77 +1,81 @@
...
+{minimum_otp_vsn, "21.3"}.
+
 {erl_opts, [
-    warnings_as_errors,
     debug_info,
     warn_export_vars,
     warn_unused_import,
+    warnings_as_errors,
+    {i, "src"},
     {parse_transform, stacktrace_transform}
```
Comment (g-andrade) [inline rebar.config]: Should we use this opportunity to get rid of `stacktrace_transform`? :)

Comment (lrascao) [reply rebar.config]: yes please

Comment (paulo-ferraz-oliveira) [reply rebar.config]: We can, it wasn't in my plans to make it any more than OTP 24 compatible.

Comment (paulo-ferraz-oliveira) [reply rebar.config]: d7c87a2

Comment (g-andrade) [reply rebar.config]: [CI failed](https://github.com/miniclip/lhttpc/pull/24/checks?check_run_id=2023862027).

Comment (paulo-ferraz-oliveira) [reply rebar.config]: And why not? 😄 I always get this, when I try to steer away from a pull request's initial scope. It's OK to take `stacktrace_compat` out, but if it wasn't done and CI was passing, it surely wasn't being harmful and could be done at a later stage.

Comment (g-andrade) [reply rebar.config]: > could be done at a later stage.

As long as we have a process for that, sure. Otherwise it will be forgotten.

**Pull request merged**